### PR TITLE
Workaround for "Cannot start FRR on Fedora and Arch using kathara/frr image"

### PR DESCRIPTION
**Pull request by buonhobo**
Added workaround for https://github.com/KatharaFramework/Docker-Images/issues/10

### CODE REVIEW

File: frr/Dockerfile [hunk 1]
```diff
@@ -13,3 +13,5 @@ RUN sed -i -e 's/service/no service/' /etc/frr/vtysh.conf \
 
 RUN apt clean && \
     rm -rf /tmp/* /var/lib/apt/lists/* /var/tmp/*
+
+COPY workaround.conf /etc/systemd/system/frr.service.d/workaround.conf
```
Comment (Skazza94) [inline frr/Dockerfile]: Hi @BuonHobo,
can you rename the `workaround.conf` file into `su_frr.conf`?

Thanks,
Mariano.

**Review by Skazza94 — CHANGES_REQUESTED** [review]

File: frr/Dockerfile (hunk 1 above)
Comment (buonhobo) [reply frr/Dockerfile]: Done!

**Review by Skazza94 — APPROVED** [review]

**Comment by Skazza94** [comment]
Hi @BuonHobo,
PR looks fine! 

We will release a new Docker image with the fix really soon!

Thanks for contributing,
Mariano.

**Comment by awaum** [comment]
Hello,

`$ cat /etc/fedora-release 
Fedora release 39 (Thirty Nine)
$ docker -v
Docker version 26.0.1, build d260a54
$ kathara -v
Current version: 3.7.0
`
I installed the fix in my running container, however:
image

Please advise.

Best regards, Armin

**Comment by tcaiazzi** [comment]
Hi @awaum, 

Sorry for the late reply, I missed your comment 😇 

I just tried the workaround, and it works on my system.  

Can you explain the steps you follow? You can build the image directly from the repository, using the command: 
```
make frr
```

I also noticed that we never upload the workaround to the Docker Hub 😇  

We will do it soon!

**Pull request merged**